client/src/app/slides/motions/motion-block/motion-block-slide.component.ts

```diff
@@ -25,10 +25,10 @@ export class MotionBlockSlideComponent extends BaseSlideComponent<MotionBlockSli
         return StateCssClassMapping[motion.recommendation.css_class] || '';
     }
 
-    public getStateLabel(motion: MotionBlockSlideMotionRepresentation): string {
+    public getRecommendationLabel(motion: MotionBlockSlideMotionRepresentation): string {
         let recommendation = this.translate.instant(motion.recommendation.name);
         if (motion.recommendation_extension) {
-            recommendation += ' ' + motion.recommendation_extension;
+            recommendation += ' ' + this.motionRepo.solveExtensionPlaceHolder(motion.recommendation_extension);
```

Comment:
@emanuelschuetze This does not work in slides. You do not have viewModels loaded. THis needs to be parsed on the server.